compiler/arser/include/arser/arser.h

```diff
@@ -650,6 +692,41 @@ template <typename T> std::vector<T> Arser::get_impl(const std::string &arg_name
   return data;
 }
 
+// Accumulated arguments with vector type (e.g., STR_VEC)
+template <typename T>
+std::vector<std::vector<T>> Arser::get_impl(const std::string &arg_name,
+                                            std::vector<std::vector<T>> *)
+{
+  auto arg = _arg_map.find(arg_name);
+  if (arg == _arg_map.end())
+    throw std::runtime_error("Invalid argument. "
+                             "There is no argument you are looking for: " +
+                             arg_name);
+
+  if (not arg->second->_is_accumulated)
+    throw std::runtime_error(
+      "Type mismatch. "
+      "You called get using a type different from the one you specified."
+      "Accumulated argument is returned as std::vector of the specified type");
```

Comment:
Last sentence seems not necessary because user already give the `std::vector<std::vector<T>>`.
```suggestion
      "Type mismatch. "
      "You called get using a type different from the one you specified.");
```